metadata.tvshows.thetvdb.com.v4.python/test_thetvdb.py

```diff
@@ -0,0 +1,46 @@
+import unittest
```

Comment:
Please move tests outside the addon directory

Reply:
Ok @romanvm . I just moved the tests out of the addon package.
See commit: https://github.com/thetvdb/metadata.tvshows.thetvdb.com.v4.python/pull/12/commits/5d4b85068aab1ff6dae1dad06c312b1f48494647 

Please, review it when you get a chance and let me know if you have any further concern.